md_api/src/main/java/com/wipro/ats/bdre/md/api/TermJob.java

```diff
@@ -35,6 +36,10 @@
  * Created by arijit on 12/8/14.
  */
 public class TermJob extends MetadataAPIBase {
+    public TermJob() {
+        AutowireCapableBeanFactory acbFactory = getAutowireCapableBeanFactory();
+        acbFactory.autowireBean(this);
+    }
 
     private static final Logger LOGGER = Logger.getLogger(TermJob.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)